src/types.ts

```diff
@@ -0,0 +1,25 @@
+export enum SortType {
+  Length = 'length',
+  Alphabet = 'alphabet',
+  Default = '',
+}
+
+export type State = {
```

Comment:
```suggestion
export interface State = {
```
If you can use the interface best way use it instead type